deluge/common.py

```diff
@@ -44,6 +45,13 @@
 
     os.environ['SSL_CERT_FILE'] = where()
 
+# On windows sock.if_nameindex() is only supported on 3.8+ and does not return a uuid.
```

Comment:
Can you put this comment in the commit message

Reply:
for some reason the git ui im using is not letting me.

Reply:
nvm I got it to work.